# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

bool emergency_brake
bool obstacle_on_right_side
bool obstacle_on_left_side
bool obstacle_behind
int8[] US_id
int32[] LIDAR_id
float32[] US_distance
float32[] LIDAR_distance

